akka-docs/src/main/paradox/scala/scala-compat.md

```diff
@@ -0,0 +1,37 @@
+# Java 8 and Scala Compatibility
```

Comment:
Moving this from `paradox/java/scala-compat.md `so that we consistently have .md files under the `paradox/scala` directory